### ISSUE TITLE
Add all padding modules to the docs

### ISSUE THREAD

**Issue by alexandresablayrolles** [pytorch/pytorch#1127]
Is there a way to use reflection padding instead of zero padding with nn.Conv2d ? I see there is a [legacy implementation](https://github.com/pytorch/pytorch/blob/master/torch/legacy/nn/SpatialReplicationPadding.py) but the option is not in nn.Conv2d.
Thanks,

**Comment by apaszke** [comment]
[The modules](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/padding.py#L6-L30) are in `nn`, but they're not exposed in the docs.

**Comment by alexandresablayrolles** [comment]
Thanks!
